schemas/material_schema.json

```
@@ -185,6 +185,18 @@
             "type": "string",
             "pattern": ".*_brdf\\.xompt$"
           }
+        },
+        "customProperties" : {
+          "type" : "array",
+          "description" : "Custom properties in this material.",
+          "items" : {
+            "type" : "object",
+            "properties" : {
+            },
+            "additionalProperties" : {
+              "type" : "object"
+            }
+          }
         }
```

```suggestion
    "customProperties": {
      "type": "object",
      "description" : "Non-standardized material properties for custom tools or tool chains.",
      "propertyValues": {
        "type": "string"
      }
    }
```

Slight adjustment to be harmonized with the definition in the asset file.
```

---

Thank you, Clemens-san, for checking the content and suggesting corrections.

Excuse me, but what is the definition of json schema propertyValues?
No propertyValues were found in the json schema specification.
[JSON Schema Specification](https://json-schema.org/specification)

Anyway,
- In the asset file, declare a descriptive extension name.
- In the material file, you can set multiple detailed parameters and their values included in the declared extension.
This parameter value can be set to either a string or a number.

Is it correct to understand that?
If my understanding is not wrong, I agree with your suggestion.

---

"propertyValues" is basically the same as the "properties" field in your proposal. It is just a name. We can also change it back to "properties".

> In the asset file, declare a descriptive extension name

In the meeting I proposed that we make it more generic. So you can use the custom properties to describe what kind of extension you use in the materials, but you can also use it for other custom properties directly in the asset file.

---

Clemens-san, Thank you for the explanation.
It seems that I missed what you explained during the meeting. I'm sorry.
I had understand your opinion.
I agree with your amendment suggestion.